python/CMakeLists.txt

```
@@ -83,6 +86,15 @@ endif()
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/__init__.py.in"
   "${CMAKE_CURRENT_BINARY_DIR}/__init__.py-install" @ONLY)
 
+# Stubs
+if (F3D_BINDINGS_PYTHON_GENERATE_STUBS)
+  add_custom_command(
+    TARGET pyf3d
+    POST_BUILD
+    COMMAND python ${CMAKE_CURRENT_SOURCE_DIR}/generate_stubs.py --into=${f3d_binary_dir}
```

```suggestion
    COMMAND PYTHONPATH="" python ${CMAKE_CURRENT_SOURCE_DIR}/generate_stubs.py --into=${f3d_binary_dir}
```
for some reason that works for me locally (linux)